TESTS/netsocket/dns/dns_tests.h

```diff
@@ -74,7 +74,13 @@ namespace dns_global {
 #ifdef MBED_GREENTEA_TEST_DNSSOCKET_TIMEOUT_S
 static const int TESTS_TIMEOUT = MBED_GREENTEA_TEST_DNSSOCKET_TIMEOUT_S;
 #else
-static const int TESTS_TIMEOUT = 14 * 60;
+#define MESH 3
+#define WISUN 0x2345
+#if MBED_CONF_TARGET_NETWORK_DEFAULT_INTERFACE_TYPE == MESH && MBED_CONF_NSAPI_DEFAULT_MESH_TYPE == WISUN
```

Comment:
&lowPAN could have the same timeouts as they should have the same network formation times.

But This is also ok